README.md

```diff
@@ -1 +1,43 @@
-# aws-infrastructure-as-code
```

Comment:
Instead of AWS infra, you can specify the name like **AWS EC2 as infrastructure using terraform**